terminus-ui/selection-list/src/selection-list.component.spec.ts

```diff
@@ -965,4 +965,24 @@ describe(`TsSelectionListComponent`, function() {
     expect(fixture.componentInstance.clicked).toHaveBeenCalled();
   }));
 
+  // Note: Simply dispatching the arrow down key on the trigger repeatedly did not wor
+  test.todo(`should update panel scroll position when focusing an out-of-view option`);
+
+  test(`should focus first option when the options collection changes`, fakeAsync(() => {
+    const fixture = createComponent(testComponents.Basic);
+    fixture.detectChanges();
+    const instance = getSelectionListInstance(fixture);
+
+    expect(instance.panel.keyManager.activeItemIndex).toEqual(-1);
+
+    const input = getSelectionListInput(fixture);
+    const states = fixture.componentInstance.states;
+    const name = states[3].name.substring(0, 2);
+    typeInElement(name, input);
+    tick(1000);
+    fixture.detectChanges();
+
+    expect(instance.panel.keyManager.activeItemIndex).toEqual(0);
```

Comment:
This however doesn't test using arrow key that option list should scroll when list is longer than viewport height - do notice that arrow key doesn't work.... we do have other specs that using arrow down key, right? wonder how they're different..

Reply:
I wouldn't expect it to? This test is called `should focus first option when the options collection changes` and that is all it tests. There is a TODO test above this to cover that called `should update panel scroll position when focusing an out-of-view option`

Reply:
Okay, I didn't see a todo and that's all the specs you added so just wanted to point it out...